<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QWidget" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background:#00000000;}
QWidget#bk2{background:#A0000000;}
QWidget#bk{border:1pxsolid#999999;}

QWidget#miniview{background:#80333333;}
QWidget#rectBorder{border:1pxsolid#FFFFFF;}

QPushButton#btnClose{border:none;border-radius:13px;background:#A0000000;image:url(:/Images/close_hover.png);}
QPushButton#btnClose:hover{background:#F25561;border-image:url(:/Images/close_hover.png);}
QPushButton#btnClose:pressed{background:#E74450;border-image:url(:/Images/close_hover.png);}

QLabel#ratetext{font-size:15px;border:1pxsolid#d7d8d9;border-radius:4px;background-color:rgb(255,255,255);}

QLabel#visiablerect{border:1pxsolid#000000;}

QPushButton#pre{border-image:url();}
QPushButton#pre:hover{border-image:url(:/Images/pre_n.png);}
QPushButton#pre:pressed{border-image:url(:/Images/pre_n.png);}

QPushButton#next{border-image:url();}
QPushButton#next:hover{border-image:url(:/Images/next_n.png);}
QPushButton#next:pressed{border-image:url(:/Images/next_n.png);}

QWidget#boolbar{background:#80333333;border-top-right-radius:4px;border-top-left-radius:4px;}

QPushButton#normalsize{border:none;image:url(:/Images/icon/imagebar/normalsize_normal.png);}
QPushButton#normalsize:hover ,QPushButton#normalsize:pressed {image:url(:/Images/icon/imagebar/normalsize_hover.png);}


QPushButton#openfolder{border:none;image:url(:/Images/icon/imagebar/folder_normal.png);}
QPushButton#openfolder:hover ,QPushButton#openfolder:pressed {image:url(:/Images/icon/imagebar/folder_hover.png);}

QPushButton#bigger{border:none;image:url(:/Images/icon/imagebar/bigger_normal.png);}
QPushButton#bigger:hover ,QPushButton#bigger:pressed {image:url(:/Images/icon/imagebar/bigger_hover.png);}

QPushButton#smaller{border:none;image:url(:/Images/icon/imagebar/smaller_normal.png);}
QPushButton#smaller:hover ,QPushButton#smaller:pressed {image:url(:/Images/icon/imagebar/smaller_hover.png);}

QPushButton#saveas{border:none;image:url(:/Images/icon/imagebar/saveas_normal.png);}
QPushButton#saveas:hover ,QPushButton#saveas:pressed {image:url(:/Images/icon/imagebar/saveas_hover.png);}

QPushButton#turnright{border:none;image:url(:/Images/icon/imagebar/turnright_normal.png);}
QPushButton#turnright:hover ,QPushButton#turnright:pressed {image:url(:/Images/icon/imagebar/turnright_hover.png);}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bk2" native="true">
     <widget class="QWidget" name="bk" native="true">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>430</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="webview" name="webRemoteUrl" native="true">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>300</width>
         <height>200</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="ratetext">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>54</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="miniview" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>150</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <widget class="QLabel" name="miniimg">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="rectBorder" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>24</width>
          <height>32</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="visiablerect">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="boolbar" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="normalsize">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>正常大小</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openfolder">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>打开所在文件夹</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveas">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>另存为</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="turnright">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>向右翻转90°</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bigger">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>放大</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="smaller">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>缩小</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pre">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="next">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>300</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClose">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>300</y>
         <width>27</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lbScledPixmap">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>lbScledPixmap</zorder>
     <zorder>bk</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>webview</class>
   <extends>QWidget</extends>
   <header>webview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="localman.qrc"/>
 </resources>
 <connections/>
</ui>
